app/models/plant_population.rb

```diff
@@ -29,6 +29,13 @@ class PlantPopulation < ActiveRecord::Base
 
   include Filterable
 
+  validates :name,
+            presence: true,
+            allow_blank: true
+
+  validates :canonical_population_name,
+            presence: true
```

Comment:
All existing records have what looks like a proper value in this field, but you're right - it's not NN. I will remove this validation.